config/samples/backends/hpe/3par/fc/backend.yaml

```diff
@@ -0,0 +1,33 @@
+# To be able to use this sample it is necessary to:
+# - Have a 3PAR backend with FC support
+# - Have the 3PAR credentials in cinder-volume-3par-secrets.yaml
+# - Having multpathd running on the host (done by this sample)
+# - Build a cinder-volume container with the 3parclient cinder driver dependency, make it available in a registry accessible by OpenShift, and set its location in this file's containerImage
+#
+# There is a sample Dockerfile in the parent directory showing how the cinder-volume container in this containerImage was created.
+# The correct way to configure and run the daemons is using `MachineConfig` as shown in `iscsid.yaml` and `multipathd.yaml`, like this sample does.
+#
+# ATTENTION: After applying this OpenShift nodes will reboot, because of the `MachineConfig` changes and will take a while to recover.
+
+apiVersion: core.openstack.org/v1beta1
+kind: OpenStackControlPlane
+metadata:
+  name: openstack
+spec:
+  cinder:
+    template:
+      cinderVolumes:
+        hpe3par-fc:
+          networkAttachments:
+          - storage
+          containerImage: quay.io/geguileo/podified-antelope-openstack-cinder-volume-3par:current-podified
```

Comment:
We probably need to provide instructions (or links to them) for building a pushing these custom c-vol images. Users will need to update L23 with the URL to the image they built and pushed.

Reply:
I didn't provide step by step instructions, but I did mention it in the `README.md` file, though on second look it was too brief. I'll add a couple more words there.